_topic/android-restricted-settings.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+title: "Restricted settings on Android."
+prompt: "Learn how to update restricted settings on Android for the Pebble app to work properly on Android 13+."
+keywords: "android app settings notifications"
+date:   2023-01-03 06:53:00
+author: "BlockArchitech"
+hideFirstStepNumber: false
+osSpecific: true
+os: android
+---
+
+# 1. Restricted Settings
+On Android 13 and higher, restricted settings prevents the Pebble app from working properly.
+
+
+![](https://raw.githubusercontent.com/BlockArchitech/temporary-images-repo/master/restricted.png)
+
+# 2. Open Settings
+
+Open the settings app on your device.
+
+
+![](https://raw.githubusercontent.com/BlockArchitech/temporary-images-repo/master/open_settings.png)
+
+# 3. Apps menu
+
+Open the apps menu, and tap "All Apps"
+
+
+![](https://raw.githubusercontent.com/BlockArchitech/temporary-images-repo/master/apps_page.png)
+
+# 4. Search for the pebble app
+
+Search for the pebble app in the apps list.
+
+
+![](https://raw.githubusercontent.com/BlockArchitech/temporary-images-repo/master/search_pebble.png)
+
+# 5. Open the hamburger menu
+
+Open the hamburger menu, and tap "Enable Restricted Settings". You'll have to enter your screen lock for this.
+
+![](https://raw.githubusercontent.com/BlockArchitech/temporary-images-repo/master/hamburger.png)
```

Comment:
Could we put a circle or something around the hamburger menu to make it super obvious?
(also pls move images to /images)

Reply:
Didn't see /images, I'll get those moved over.